tests/ecosystem/upgrade/test_upgrade.py

```diff
@@ -88,3 +96,22 @@ def test_check_mon_pdb_post_upgrade():
     # The PDB values are considered from OCS 4.5 onwards.
     assert disruptions_allowed == 1, "Mon Disruptions_allowed count not matching"
     assert min_available_mon == 2, "Minimum available mon count is not matching"
+
+
+@post_ocp_upgrade
+@post_ocs_upgrade
+@pytest.mark.polarion_id("OCS-2524")
```

Comment:
existing test test_check_mon_pdb_post_upgrade.py  checks mon pdb.
Since both mon and osd pdb values are same and has same implementation, can we combine both osd and mon pdb checks post upgrade in a single test?

Reply:
DOne